appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -805,31 +807,33 @@ public RpcResult build(User user, long projectId, String nonce, String target,
   }
 
   String buildErrorMsg(String exceptionName, URL buildURL, String userId, long projectId) {
-    return "Request to build failed with " + exceptionName 
-      + ", user=" + userId + ", project=" + projectId 
-      + ", build URL is " + (buildURL != null ? buildURL : "null") + " [" 
+    return "Request to build failed with " + exceptionName
+      + ", user=" + userId + ", project=" + projectId
+      + ", build URL is " + (buildURL != null ? buildURL : "null") + " ["
       + (buildURL != null ? buildURL.toString().length() : "n/a") + "]";
   }
 
   // Note that this is a function rather than just a constant because we assume it will get
   // a little more complicated when we want to get the URL from an App Engine config file or
   // command line argument.
   private String getBuildServerUrlStr(String userName, String userId,
-    long projectId, boolean secondBuildserver, String fileName)
-      throws UnsupportedEncodingException, EncryptionException {
-    return "http://" + (secondBuildserver ? buildServerHost2.get() : buildServerHost.get()) +
-      "/buildserver/build-all-from-zip-async"
-      + "?uname=" + URLEncoder.encode(userName, "UTF-8")
-      + (sendGitVersion.get()
-        ? "&gitBuildVersion="
-        + URLEncoder.encode(GitBuildId.getVersion(), "UTF-8")
-        : "")
-      + "&callback="
-      + URLEncoder.encode("http://" + getCurrentHost() + ServerLayout.ODE_BASEURL_NOAUTH
-        + ServerLayout.RECEIVE_BUILD_SERVLET + "/"
-        + Security.encryptUserAndProjectId(userId, projectId)
-        + "/" + fileName,
-        "UTF-8");
+    long projectId, boolean secondBuildserver, String fileName, boolean isAab)
+      throws EncryptionException {
+    UriBuilder uriBuilder = new UriBuilder(
+        "http://" +
+        (secondBuildserver ? buildServerHost2.get() : buildServerHost.get()) +
+        "/buildserver/build-all-from-zip-async"
+    )
```

Comment:
Move this trailing parens to the previous line.

Reply:
Intellij recommended also to increase the indent by 4, like this:

```
    UriBuilder uriBuilder = new UriBuilder(
        "http://" +
            (secondBuildserver ? buildServerHost2.get() : buildServerHost.get()) +
            "/buildserver/build-all-from-zip-async")
        .add("uname", userName)
        .add("callback", "http://" + getCurrentHost() + ServerLayout.ODE_BASEURL_NOAUTH +
            ServerLayout.RECEIVE_BUILD_SERVLET + "/" +
            Security.encryptUserAndProjectId(userId, projectId) + "/" +
            fileName)
```

Should I also apply it?